components/StreamField/StreamField.module.scss

```diff
@@ -1,5 +1,15 @@
 .container {
-    max-width: 800px;
+    @include media-query(large) {
+        margin: 0 calc(5rem + 80px);
```

Comment:
I've been noticing a lot of these sort of hard coded values `rem` or `px`. This isn't an issue.

What do you think about adding these into `variables.scss` and referring to them as variables in the future?

Reply:
I did this mainly to keep the topic page hero and the streamfield content always aligned, so it's somehow specific to this file.

I agree that having variables names would be more readable but I'm afraid to break the styling on some parts without noticing it.